src/utils.js

```diff
@@ -25,13 +26,23 @@ export const parseInclude = request => {
   const models = getModels(request);
   if (models.isBoom) return models;
 
-  return include.map(a => {
-    const singluarOrPluralMatch = Object.keys(models).find((modelName) => {
-      const { _singular, _plural } = models[modelName];
-      return _singular === a || _plural === a;
-    });
+  return include.map(b => {
+    let a = b;
+    try {
+      if (joi.string().regex(/^\{.*?"model":.*?\}$/)) {
```

Comment:
but with this code, it won't validate the model name and the "where" object isn't it?
if the JSON string doesn't parse good so it caught as a string, but later it won't pass because crud.js have something like `joi.string().valid(...modelAssociations)`